watertap/property_models/anaerobic_digestion/adm1_reactions.py

```diff
@@ -117,15 +117,11 @@ def build(self):
             "S_bu": 0.0250,
             "S_pro": 0.0267867187876688,
             "S_ac": 0.0312221602404522,
-            "S_ch4": 0.0156255859594734,
```

Comment:
So, these deleted factors--where are they used in the Petersen matrix under a different parameter/var? The reason I ask is because I don't think we should have majority of carbon content parameters defined here, while the four values being deleted are being used somewhere, only under a different attribute or local variable. Does that make sense?